pkgs/data/fonts/lexend/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lexend";
+  version = "0.pre+date=2022-01-27";
+
+  src = fetchFromGitHub {
+    owner = "googlefonts";
+    repo = pname;
+    rev = "57e6c14e2a9b457e8376044a31525c2100297e9c";
+    sha256 = "sha256-+tPggQIO50a8kOSnOVN/MR9ZwX5xZqYVNZO79eog9QA=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    cd fonts
+    for f in *; do
+      mkdir -p $out/share/fonts/truetype/lexend/$f
+      install $f/ttf/* $out/share/fonts/truetype/lexend/$f/
+    done
```

Comment:
If we only copy files we can also move this to postFetch in fetchFromGitHub.

Reply:
I don't think it's working properly, or I'm missing something.
```nix
{ lib
, fetchFromGitHub
}:
let
  pname = "lexend";
  version = "0.pre+date=2022-01-27";
  rev = "57e6c14e2a9b457e8376044a31525c2100297e9c";
in
fetchFromGitHub {
  name = "${pname}-${version}";

  owner = "googlefonts";
  repo = pname;
  inherit rev;
  sha256 = "sha256-p1Yc+mM/felZuhpSrzltLdMcFG9ZP5riW3QmAYRFAf4=";

  postFetch = ''
    tar -xf $downloadedFile --strip=1
    mkdir -p $out/share/{fonts/truetype,doc}/lexend

    for f in fonts/*; do
      mkdir -p $out/share/fonts/truetype/lexend/$f
      cp -r fonts/$f/ttf/* $out/share/fonts/truetype/lexend/$f/
    done

    cp README.md documentation $out/share/doc/lexend
  '';

  meta = with lib; {
    homepage = "https://www.lexend.com";
    description = "A variable font family designed to aid in reading proficiency";
    license = licenses.ofl;
    platforms = platforms.all;
    maintainers = with maintainers; [ fufexan ];
  };
}
```

Reply:
no idea